### PR TITLE
Adding AI2 Logos into varnish-mui

### PR DESCRIPTION
**Pull request by arnavic**
Porting over the AI2 Logos from Varnish into Varnish Mui, with slight modifications to render the docs in the correct Material UI manner. 

Screenshots of Varnish-MUI Logos Page: 

image

image

image

image

### CODE REVIEW

**Comment by arnavic** [comment]
this addresses issue #17

**Review by codeviking — APPROVED** [review]
lgtm!

File: docs/data/material/varnish/logos/logos.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+{{"demo": "ProjectsLogosDemo.js"}}
+
+## API
+
+### AI2 Logo
+
+| Property    | Description                | Type                         | Default   |
+| ----------- | -------------------------- | ---------------------------- | --------- |
+| size        | sets size of logo          | 'micro' \| 'default' \| 'lg' | 'default' |
```
Comment (jonborchardt) [inline docs/data/material/varnish/logos/logos.md:45]: can you please convert to use 
xs, sm, md, lg, and xl
to be more like material?

File: docs/data/material/varnish/logos/ProjectsLogosDemo.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { logos } from '@allenai/varnish';
+
+export default function Demo() {
+    return (
+        <span>
+            <div>AllenNLP</div>
```
Comment (jonborchardt) [inline docs/data/material/varnish/logos/ProjectsLogosDemo.tsx:6]: please use a stack
https://mui.com/material-ui/react-stack/

File: docs/data/material/varnish/logos/logos.md (hunk 1 above)
Comment (arnavic) [reply docs/data/material/varnish/logos/logos.md:45]: sure! i think sm, md, lg seems like the right ones to use for the 3 logo sizes?

Comment (jonborchardt) [reply docs/data/material/varnish/logos/logos.md:45]: yep.
we should proactively try to align with material whenever possible

**Pull request merged**